storage/src/test/java/org/apache/kafka/tiered/storage/TieredStorageTestContext.java

```diff
@@ -309,7 +310,5 @@ public void printReport(PrintStream output) {
 
     @Override
     public void close() throws IOException {
-        Utils.closeAll(producer, consumer);
```

Comment:
For my understanding, why are these safe to remove?

Reply:
clients are created using harness (harness.createConsumer), so they are auto-closed on tear-down. This is one of the follow-up [comment](https://github.com/apache/kafka/pull/14116#discussion_r1294668715) from PR #14116.

Reply:
Okay, do you need the overriden method if it has no implementation?

Reply:
Left it to close any other items in future if required.